src/main/java/Bundle_fr.properties

```diff
@@ -0,0 +1,2021 @@
+harvestclients.newClientDialog.url.required=Une adresse valide de serveur � moissonner est requise.
+harvestclients.newClientDialog.url.invalid=URL non valide. Impossible d'�tablir une connexion et recevoir une r�ponse valide du serveur.
+harvestclients.newClientDialog.url.noresponse=Impossible d'�tablir la connexion avec le serveur.
+harvestclients.newClientDialog.url.badresponse=R�ponse non valide de la part du serveur.
+harvestclients.newClientDialog.dataverse=Dataverse local
+harvestclients.newClientDialog.dataverse.tip=Dataverse qui h�bergera les ensembles de donn�es r�colt�s � partir de cette ressource distante.
+harvestclients.newClientDialog.dataverse.menu.enterName=Saisir l'alias du dataverse
+harvestclients.newClientDialog.dataverse.menu.header=Nom du dataverse (affiliation), alias
+harvestclients.newClientDialog.dataverse.menu.invalidMsg=Aucun r�sultat
+harvestclients.newClientDialog.dataverse.required=Vous devez s�lectionner un dataverse existant pour ce client de moissonnage.
+harvestclients.newClientDialog.step2=�tape 2 de 4 - Format
+harvestclients.newClientDialog.oaiSets=Ensemble OAI
+harvestclients.newClientDialog.oaiSets.tip=Ensembles moissonnables offerts par ce serveur OAI.
+harvestclients.newClientDialog.oaiSets.noset=Aucun
+harvestclients.newClientDialog.oaiSets.helptext=En s�lectionnant �\u00A0Aucun\u00A0� le moissonnage se fera sur l'ensemble par d�faut d�fini par le serveur. Fr�quemment il s'agit de l'enti�ret� du contenu de tous les sous-ensembles.
+harvestclients.newClientDialog.oaiSets.helptext.noset=Ce serveur OAI ne prend pas en charge les ensembles s�lectionn�s. L'ensemble du contenu propos� par le serveur sera moissonn�.
+harvestclients.newClientDialog.oaiMetadataFormat=Format des m�tadonn�es
+harvestclients.newClientDialog.oaiMetadataFormat.tip=Formats de m�tadonn�es offerts par le serveur distant.
+harvestclients.newClientDialog.oaiMetadataFormat.required=Veuillez s�lectionner le format de m�tadonn�es � utiliser pour le moissonnage de ce d�p�t.
+harvestclients.newClientDialog.step3=�tape 3 de 4 - Planifier
+harvestclients.newClientDialog.schedule=P�riodicit�
+harvestclients.newClientDialog.schedule.tip=Programmer le moissonnage pour qu'il s'ex�cute automatiquement de fa�on quotidienne ou hebdomadaire.
+harvestclients.newClientDialog.schedule.time.none.helptext=Ne pas sp�cifier de p�riodicit� de moissonnage de sorte que l'ex�cution se fera sur demande seulement.
+harvestclients.newClientDialog.schedule.none=Aucun
+harvestclients.newClientDialog.schedule.daily=Quotidiennement
+harvestclients.newClientDialog.schedule.weekly=Hebdomadairement
+harvestclients.newClientDialog.schedule.time=Horaire
+harvestclients.newClientDialog.schedule.day=Jour
+harvestclients.newClientDialog.schedule.time.am=a.m.
+harvestclients.newClientDialog.schedule.time.pm=p.m.
```

Comment:
In French, the hour is written according to the 24-hour time system. Hence abbreviations like a.m. and p.m. and all their derivatives (AM, PM, am, pm, etc.) that come from the Latin expressions ante meridiem ("before noon") and post meridiem ("after noon") should not be used.

However since we must provide a translation for the 2 harvestclients.newClientDialog.schedule.time.* variables, I would go for either:

1) a.m. and p.m. forms, which seams to be the accepted forms in English (ref.: https://www.btb.termiumplus.gc.ca/tpv2guides/guides/favart/index-eng.html?lang=eng&lettr=indx_titls&page=9qo3G9wQtvTo.html) 

or

2) not use abbreviations and use the accepted workaround instead:

harvestclients.newClientDialog.schedule.time.am=en avant-midi
harvestclients.newClientDialog.schedule.time.pm=en après-midi


A better solution, that goes beyond translation issues, would be to have Dataverse use a 24-hour time system for this menu. I believe that this would also be more acceptable in a perspective of internationalization.